src/runtime-rs/crates/shim/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+name = "shim"
+version = "0.1.0"
+authors = ["The Kata Containers community <kata-dev@lists.katacontainers.io>"]
+description = "Containerd shim runtime for Kata Containers"
+keywords = ["kata-containers", "shim"]
+repository = "https://github.com/kata-containers/kata-containers.git"
+license = "Apache-2.0"
+edition = "2018"
+
+[[bin]]
+name = "containerd-shim-kata-v3"
```

Comment:
Here `v2` stands for containerd runtime v2 api. It will not work with `containerd` if changed to `v3`

Reply:
It may work:

https://github.com/containerd/containerd/blob/2a3f1094a4684222ffc1078a171fda9bc650eac5/runtime/v2/shim/util.go#L83-L93

Reply:
Ops, I was wrong. Thanks @liubin for the clarification

Reply:
Thanks for the review, use containerd-shim-rund-v2